.tests/integration/units.tsv

```diff
@@ -1,4 +1,4 @@
-sample	type	machine	platform	flowcell	lane	barcode	fastq1	fastq2	adapter
-NA12878	T	NDX550407_RUO	NextSeq	HKTG2BGXG	L001	ACGGAACA+ACGAGAAC	fastq/NA12878_fastq1.fastq.gz	fastq/NA12878_fastq2.fastq.gz	ACGT,ACGT
-NA12879	N	NDX550407_RUO	NextSeq	HKTG2BGXG	L001	ACGGAACA+ACGAGAAC	fastq/NA12878_fastq1.fastq.gz	fastq/NA12878_fastq2.fastq.gz	ACGT,ACGT
-NA12879	R	NDX550407_RUO	NextSeq	HKTG2BGXG	L001	ACGGAACA+ACGAGAAC	fastq/NA12878_fastq1.fastq.gz	fastq/NA12878_fastq2.fastq.gz	ACGT,ACGT
+sample	type	machine	platform	flowcell	lane	barcode	fastq1	fastq2	adapter	bam	vcf	gvcf	cnv_vcf	
```

Comment:
Why do we still need the extra headers?

Reply:
Some problem with the version used with reference! I tried with develop and then I could have a unit file without bam and vcf. Could we do a release of reference and start using that?